code/modules/tgui/modules/volume_mixer.dm

```diff
@@ -0,0 +1,38 @@
+/datum/ui_module/volume_mixer
+	name = "Volume Mixer"
+
+/datum/ui_module/volume_mixer/ui_interact(mob/user, ui_key = "main", datum/tgui/ui = null, force_open = FALSE, datum/tgui/master_ui = null, datum/ui_state/state = GLOB.always_state)
+	ui = SStgui.try_update_ui(user, src, ui_key, ui, force_open)
+	if(!ui)
+		ui = new(user, src, ui_key, "VolumeMixer", name, 400, 300, master_ui, state)
```

Comment:
Any way to make the UI height scale based on amount of faders? (Clamp to 600 height obviously)